### ISSUE TITLE
Color preview not shown for CSS variables with static fallback color values.

### ISSUE THREAD

**Issue by minht11** [tailwindlabs/tailwindcss-intellisense#680]
**What version of Tailwind CSS IntelliSense are you using?**
v0.9.1

**What version of Tailwind CSS are you using?**
v3.2.4

**Tailwind config**

```js
/** @type {import('tailwindcss').Config} */
module.exports = {
	content: ['./src/**/*.{html,js,svelte,ts}'],
	theme: {
		colors: {
			// No preview shown, despite fallback value being used.
			primary: `rgba(var(--color-primary, 100, 100, 100), <alpha-value>)`,
			// Works as expected.
			secondary: `rgba(100, 100, 100, <alpha-value>)`,
		}
	},
}
```

**Describe your issue**
Color preview is not working for color tokens using css custom variables with statically recognisable fallback color values.

<img width="901" alt="image" src="https://user-images.githubusercontent.com/8914032/207727147-e9c37f88-b6d9-46c9-8447-2059da503f89.png">

Related to https://github.com/tailwindlabs/tailwindcss-intellisense/issues/415

**Comment by bradlc** [comment]
Going to close this as a duplicate of #604. We will have a new release out this week with the fix 👍

**Comment by jesspinklet** [comment]
> Going to close this as a duplicate of #604. We will have a new release out this week with the fix 👍

Is this fixed on v0.9.5? Im still having this issue
